nixos/modules/services/security/kanidm.nix

```diff
@@ -502,13 +502,13 @@ in
               };
 
               originUrl = mkOption {
-                description = "The origin URL of the service. OAuth2 redirects will only be allowed to sites under this origin. Must end with a slash.";
+                description = "The redirect URL of the service. By default this needs to exactly match the OAuth2 redirect target. When disabling strict-redirect-uri kanidm will only match by origin.";
```

Comment:
1.4 requires strict redirect URIs, so this comment confuses me. kanidm won't even start if one or more clients are not enabled for strict.

Reply:
That's not true, this is only relevant when upgrading from 1.3 -> 1.4. Afterwards the setting can be disabled on each oauth2-resource-server as required. While I didn't yet personally encounter an application that requires this to be disabled, it might be a good option to have.

Reply:
No particular reason except that I just copied the attribute names kanidm [uses internally](https://github.com/kanidm/kanidm/blob/dfbcfa865fbaea6081a7a21135ccee34ed0ede87/proto/src/attribute.rs#L122-L123).

But I think redirect-url is the definitely a more descriptive name for what it represents. I can change this to `redirectUrl` later if you like, and while we are at it we should maybe also change `originLanding` to `landingUrl`. But this will require an update to the provisioning application, too so I'll have to update that first.

Reply:
Well. I think we misunderstood, while you are able to change the strict-redirect-uri setting as of 1.4 it doesn't work anymore and will always be enabled.
I've changed the description.